Source/WebCore/platform/graphics/cocoa/FontCacheCoreText.cpp

```diff
@@ -967,7 +967,7 @@ void FontCache::platformInvalidate()
     platformShouldEnhanceTextLegibility() = _AXSEnhanceTextLegibilityEnabled();
 }
 
-static RetainPtr<CTFontRef> fontWithFamilySpecialCase(const AtomString& family, const FontDescription& fontDescription, float size, AllowUserInstalledFonts allowUserInstalledFonts)
+static RetainPtr<CTFontDescriptorRef> fontWithFamilySpecialCase(const AtomString& family, const FontDescription& fontDescription, float size, AllowUserInstalledFonts allowUserInstalledFonts)
```

Comment:
Do you think this should be called `fontDescriptionWithFamilySpecialCase` or similar? Or do we conflate "font" and "font description" enough that it doesn't matter.

Reply:
Oh, yeah, that's a good point. Distinguishing between them is kind of the whole point of this effort I'm doing. I'll rename.